native/dtrtri.go

```diff
@@ -0,0 +1,60 @@
+package native
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// Dtrtri computes the inverse of a triangular matrix, storing the result in place
+// into a. This is the BLAS level 3 version of the algorithm which builds upon
+// Dtrti2 to operate on matrix blocks instead of only individual columns.
+//
+// Dtrti returns whether the matrix a is singular or whether it's not singular.
```

Comment:
I think this could be better: "Dtrtri will not perform the inversion if the matrix is singular, and returns a boolean indicating whether the inversion was successful."

I think that covers the behaviour.